docker-compose-3-persona.yml

```diff
@@ -47,27 +47,27 @@ services:
   hydrogen-producer-postgres-hyproof-api:
     image: postgres:16.1-alpine
     container_name: hydrogen-producer-postgres-hyproof-api
-    #    ports:
-    #      - 5432:5432
+    # ports:
+    #   - 5432:5432
     volumes:
       - hydrogen-producer-hyproof-api-storage:/var/lib/postgresql/data
     environment:
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_USER=postgres
-      - POSTGRES_DB=dscp-hyproof-api
+      - 'POSTGRES_PASSWORD=postgres'
```

Comment:
is there a reason for all the diff churn adding the `'` wrappings?

Reply:
Hmmm.. i guess it is easier when every element is wrapped with quotes .. if something ends with `#fff` it is easier to tell `# something` is not a comment if you have `V=#fff` but i can put it back , no problem .. just tell me

Reply:
the main issue here is the diff churn and as this is just an escaping syntax it's something to just know if using yaml anyway